Applications/Spire/Source/Ui/ScrollBox.cpp

```diff
@@ -460,19 +460,9 @@ void ScrollBox::update_layout() {
   auto scroll_bar_size = QSize(
     m_scrollable_layer->get_vertical_scroll_bar().sizeHint().width(),
     m_scrollable_layer->get_horizontal_scroll_bar().sizeHint().height());
-  auto viewport_size = [&] {
```

Comment:
What is this change related to? Is there a task for it?

Reply:
The purpose of this code is to solve the issue of the vertical `ScrollBar` blinking when adding a new Tag. The `TagBox` has to access the view port of the `ScrollBox` and set maximum size to it. In addition, Setting the maximum size of the view port might cause the `ScrollBox` placed in a layout sometimes not to fill the entire layout (The `ScrollBox` is partially displayed in its parent).  Now I fixed the blinking by setting the vertical display policy (ON_OVERFLOW and NEVER) within the `TagBox`, so the fix was processed in the `TagBox` instead of in the `ScrollBox`, which doesn't need to hack into the `ScrollBox`. Hence, I removed these code and reverted to the original implementation of the ScrollBox.